### DISCUSSION TITLE
Show tracker status of torrent (errored / working) in torrent details panel

### DISCUSSION THREAD

**Discussion by StereotypicalCat** [jesec/flood#527] [Ideas]
Have you considered adding individual tracker statuses to the tracker page of the individual torrent?

When going into the tracker panel (right click on torrent -> torrent details -> trackers), it lists trackers by their url and their type. It would be awesome to maybe just have a third column that showed if the tracker is working or errored :)

**Comment by urwrstkn8mare** [comment]
Yeah something like this hopefully
![image](https://user-images.githubusercontent.com/42769125/163883864-7edc9873-d28e-489b-83c8-f3c85ab5b0f1.png)
